core-bundle/tests/DependencyInjection/ContaoCoreExtensionTest.php

```diff
@@ -1884,6 +1884,40 @@ public function testRegistersTheTwigTemplateExtension(): void
         $this->assertSame('contao.routing.scope_matcher', (string) $definition->getArgument(2));
     }
 
+    public function testRegistersTheCrawlResources(): void
+    {
+        $extension = new ContaoCoreExtension();
+        $extension->load([], $this->container);
+
+        $this->assertTrue($this->container->has('contao.search.escargot_factory'));
+
+        $factory = $this->container->getDefinition('contao.search.escargot_factory');
+
+        $this->assertSame([], $factory->getArgument(2));
+        $this->assertSame([], $factory->getArgument(3));
+
+        $extension->load(
+            [
+                'contao' => [
+                    'crawl' => [
+                        'additionalURIs' => [
+                            'https://examle.com',
+                        ],
+                        'defaultHttpClientOptions' => [
+                            'proxy' => 'http://localhost:7080',
+                        ],
+                    ],
+                ],
+            ],
+            $this->container
+        );
+
+        $factory = $this->container->getDefinition('contao.search.escargot_factory');
+
+        $this->assertSame(['https://examle.com'], $factory->getArgument(2));
+        $this->assertSame(['proxy' => 'http://localhost:7080'], $factory->getArgument(3));
+    }
```

Comment:
We also have to test the `contao.search.escargot_subscriber.search_index` service here, don't we?

Reply:
Not really no. This is not testing the real container but only a mock. So we wouldn't test anything at all.